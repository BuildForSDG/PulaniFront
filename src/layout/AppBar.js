import React from 'react';
import { AppBar } from 'react-admin';
import Typography from '@material-ui/core/Typography';

import { appBarStyle } from '../stylesheets';

export const PulaniAppBar = props => {
  const classes = appBarStyle();

  return (
    <AppBar {...props} className={classes.appBar}>
      <Typography
        color="inherit"
        className={classes.title}
        id="react-admin-title"
      />
      <Typography variant="h6" color="inherit">
        Pulani
      </Typography>
      <span className={classes.spacer} />
    </AppBar>
  );
};
