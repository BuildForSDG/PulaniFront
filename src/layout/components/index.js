export * from './Card';
export * from './Charts';
export * from './Grid';
