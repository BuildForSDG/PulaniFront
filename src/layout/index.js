export * from './AppBar';
export * from './Layout';

export * from './components';
