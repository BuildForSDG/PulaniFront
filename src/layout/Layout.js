import React from 'react';
import { Layout } from 'react-admin';
import { PulaniAppBar } from './AppBar';

import { PulaniTheme, layoutStyle } from '../stylesheets';

export const PulaniLayout = props => {
  const classes = layoutStyle();

  return (
    <Layout
      {...props}
      appBar={PulaniAppBar}
      theme={PulaniTheme}
      className={classes.root}
    />
  );
};
