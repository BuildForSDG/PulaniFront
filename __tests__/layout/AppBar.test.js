import React from 'react';
import { shallow } from 'enzyme';
import { PulaniAppBar } from '../../src/layout/AppBar';

describe('app module', () => {
	const wrapper = shallow(<PulaniAppBar />);

	it('that it exists', () => {
		expect(wrapper).toBeDefined();
	});

	it('that it renders without crashing', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
