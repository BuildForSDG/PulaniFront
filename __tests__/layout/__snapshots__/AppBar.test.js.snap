// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar/> module that it renders without crashing 1`] = `
<AppBar
  className="makeStyles-appBar-4"
  userMenu={
    <UserMenu
      icon={<Memo(AccountCircleIcon) />}
      label="ra.auth.user_menu"
    />
  }
>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-title-2"
    color="inherit"
    id="react-admin-title"
  />
  <WithStyles(ForwardRef(Typography))
    color="inherit"
    variant="h6"
  >
    Pulani
  </WithStyles(ForwardRef(Typography))>
  <span
    className="makeStyles-spacer-3"
  />
</AppBar>
`;
